src/azure-cli/azure/cli/command_modules/appservice/_params.py

```diff
@@ -726,6 +727,7 @@ def load_arguments(self, _):
         c.argument('ignore_stack', options_list=['--ignore-stack'], help='If true, any stack-specific defaults are ignored.', choices=['true', 'false'])
         c.argument('timeout', options_list=['--timeout'], help='Timeout for the deployment operation in milliseconds.')
         c.argument('slot', help="The name of the slot. Default to the productions slot if not specified.")
+        c.argument('track_deployment', help="If true, web app deployment status will be tracked verbosely.", arg_type=get_three_state_flag())
```

Comment:
I noticed that the PR also introduces the new argument for the command `az functionapp deploy`, but it is not mentioned in the PR description. Could you please update the PR description to include this information?

Reply:
Not including this flag for `functionapp deploy` now. Reverted changes